go/logic/applier.go

```diff
@@ -703,8 +703,9 @@ func (this *Applier) CreateAtomicCutOverSentryTable() error {
 
 	query := fmt.Sprintf(`create /* gh-ost */ table %s.%s (
 			id int auto_increment primary key
-		) comment='%s'
+		) engine=%s comment='%s'
 		`,
+		this.migrationContext.TableEngine,
```

Comment:
`TableEngine` correlates to the 3rd placeholder in the template string, not the 1st